docs/configuration-cluster.md

```diff
@@ -169,6 +169,25 @@ This will put additional load on the data store but will ensure that dimension a
 
 How often should source schema be reloaded in ms. Default value of 0 disables periodical source refresh.
 
+**auth** 
```

Comment:
I would put this section just after "url" or "host" section. Now the authentication section is together with sourceReintrospectInterval - strange.